tests/plotting/test_actor.py

```diff
@@ -146,6 +156,29 @@ def test_actor_orientation(actor):
     actor.orientation = orientation
     assert np.allclose(actor.orientation, orientation)
 
+    # test rotation order
+    dataset = pv.Cube()
+    dataset.rotate_y(orientation[1], inplace=True)
+    dataset.rotate_x(orientation[0], inplace=True)
+    dataset.rotate_z(orientation[2], inplace=True)
+
+    actor = pv.Actor(mapper=pv.DataSetMapper(pv.Cube()))
+    actor.orientation = orientation
+    assert np.allclose(dataset.bounds, actor.bounds)
+
+
+def test_actor_origin(actor):
+    assert actor.origin == (0, 0, 0)
+    origin = (1, 2, 3)
+    actor.origin = origin
+    assert np.allclose(actor.origin, origin)
+
+
+def test_actor_length(actor):
+    assert actor.length == 1.4142135623730951
+    actor.scale = 2
+    assert actor.length == 2.8284271247461903
```

Comment:
A mathematically literate person will know what these numbers mean, but they run the risk of becoming magic numbers for some people. We recommend that you define variables once and express their meaning.

Reply:
Meant to reply here, but this should be resolved with https://github.com/pyvista/pyvista/pull/6194#discussion_r1627068640